src/pages/LaboratoryPage.tsx

```diff
@@ -1,37 +1,65 @@
-import { useParams, useNavigate } from 'react-router-dom';
-import { laboratories } from '../mocks/laboratories';
-import { Laboratory } from '../types/Laboratory.ts';
-import NavigationHeader from '../components/NavigationHeader/NavigationHeader';
+import { useParams } from 'react-router-dom';
 import LaboratoryPageContent from '@/components/LaboratoryPageComponents/LaboratoryPageContent/LaboratoryPageContent.tsx';
-import styles from '@styles/components/LaboratoryPage.module.scss';
-import ButtonBack from '@public/button_back.svg';
+import DefaultPageLayout from '@/components/DefaultPageLayout/DefaultPageLayout.tsx';
+import { useEffect, useState } from 'react';
+import { LaboratoryResponseData } from '@/DTO/LaboratoryDTO.ts';
+import { useLaboratoryProvider } from '@/providers/LaboratoryProvider/useLaboratoryProvider.ts';
 
 export default function LaboratoryPage() {
+  const [activeTab, setActiveTab] = useState('lab');
+  const [laboratory, setLaboratory] = useState<LaboratoryResponseData | null>(
+    null,
+  );
+  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
Булевые стейты лучше всего именовать с приставки is-

```suggestion
  const [isLoading, setIsLoading] = useState(true);
```